### ISSUE TITLE
tflint 0.27.0 error on AWS Tag validation

### ISSUE THREAD

**Issue by ehrhardt** [terraform-linters/tflint#1102]
I found that the rule for validating AWS is not working on tflint 0.27.0 on macOS 10.15.7 (19H524).
It does work on tflint 0.26.0.
Both uses ruleset.aws 0.3.1-bundled.

## Terraform code

```hcl
resource "aws_lb" "prod" {
  name               = "mycompany-customer-facing-elb"
  internal           = false
  load_balancer_type = "application"
  security_groups    = [aws_security_group.lb_sg.id]
  subnets            = aws_subnet.public.*.id

  enable_deletion_protection = true

  access_logs {
    bucket  = aws_s3_bucket.lb_logs.bucket
    prefix  = "prod-lb"
    enabled = true
  }

  tags = {
    Cluster = "prod"
  }
}
```
## Version

```console
% tflint --version ; echo buggy
TFLint version 0.27.0
+ ruleset.aws (0.3.1-bundled)
buggy
% ./tflint --version ; echo works
TFLint version 0.26.0
+ ruleset.aws (0.3.1-bundled)
works
%
```

## Output

```
% tflint ; echo $?
Failed to check ruleset. An error occurred:

Error: Failed to check `aws_resource_missing_tags` rule: gob: type not registered for interface: map[string]string

2
% ./tflint ; echo $?
0
%
```

**Comment by rquadling** [comment]
I'm getting this error too. With the debug enabled ...

```
2021-04-20T14:58:47.007+0100 [DEBUG] plugin.tflint: 14:58:47 aws_resource_missing_tags.go:95: [DEBUG] Walk `aws_acm_certificate.certificate.tags` attribute
Failed to check ruleset. An error occurred:

Error: Failed to check `aws_resource_missing_tags` rule: gob: type not registered for interface: map[string]string
```

And that particular file / attribute ...
```hcl
resource "aws_acm_certificate" "certificate" {
  domain_name       = "*.${var.domain}.${var.tlds["UK"].tld}"
  validation_method = "DNS"

  subject_alternative_names = [for tld in setsubtract(values(var.tlds), [
    {
      tld     = var.tlds["UK"].tld
      zone_id = var.tlds["UK"].zone_id
  }]) : "*.${var.domain}.${tld.tld}"]

  tags = merge(local.base_tags, {
    "Name" = "${var.cluster}-${var.environment}-certificate"
  })
}
```

**Comment by bendrucker** [comment]
I've added a repro case in #1104 but haven't been able to identify how this regression was introduced quite yet

**Comment by wata727** [comment]
Interesting. This issue is obvious by looking at https://github.com/terraform-linters/tflint-ruleset-aws/pull/106, but I'm not sure why v0.26 doesn't cause an error.

I tried to check if it can be reproduced in my local machine, but it seems that an error occurs even with v0.26 except for the bundled plugin. I have confirmed that the same error occurs in TFLint v0.26 + AWS ruleset v0.3.1 (not bundled).

**Comment by wata727** [comment]
Fixed in #1104 by updating bundled AWS ruleset plugin.
